react/src/components/FileBrowserModal/FileBrowserModal.tsx

```diff
@@ -14,31 +17,62 @@ const { Text } = Typography;
 const FileBrowserModal = ({
   isOpen,
   toggle: parentToggle,
+  onImported,
   allowedFileExtensions = [],
 }: FileBrowserModalProps) => {
+  const [selectedFiles, setSelectedFiles] = useState<File[]>([]);
+
   const handleClose = () => {
     parentToggle();
   };
 
+  const handleFileSelect = (files: File[]) => {
+    console.log(files);
+    setSelectedFiles(files);
+  };
+
+  const handleImport = () => {
+    console.log(selectedFiles);
```

Comment:
Do you want to leave this log statement here?

Reply:
Thanks!  Removed.